examples/two_moons_classification_sngp.pct.py

```diff
@@ -0,0 +1,228 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.14.5
+#   kernelspec:
+#     display_name: python3
+#     language: python
+#     name: python3
+# ---
+
+# %% [markdown]
+
+# # Two-moons Classification: Improved uncertainty quantification with SNGP
+
+# %% [markdown]
+# In this notebook we show how to train an [SNGP](https://arxiv.org/abs/2006.10108) model using Fortuna, showing improved
+# uncertainty estimation on the two moons dataset w.r.t it's deterministic counterpart.
+
+
+# %% [markdown]
+# ### Download Two-Moons data from scikit-learn
+# Let us first download two-moons data from [scikit-learn](https://scikit-learn.org/stable/modules/generated/sklearn.datasets.make_moons.html).
+
+# %%
+TRAIN_DATA_SIZE = 2000
+
+from sklearn.datasets import make_moons
+train_data = make_moons(n_samples=TRAIN_DATA_SIZE, noise=0.1, random_state=0)
+val_data = make_moons(n_samples=500, noise=0.1, random_state=1)
+test_data = make_moons(n_samples=500, noise=0.1, random_state=2)
+
+# %% [markdown]
+# ### Convert data to a compatible data loader
+# Fortuna helps you converting data and data loaders into a data loader that Fortuna can digest.
+
+# %%
+from fortuna.data import DataLoader
```

Comment:
A general comment that all imports should be at the top of the file to adhere to PEP rules.

Reply:
Agreed, but is this true also for the example notebooks?

Reply:
This is on me. I'm using imports in each cell of example notebooks to make it easier to understand where objects are called from. I guess it is debatable what's best practice in this case.